src/main/scala/thurloe/service/ThurloeService.scala

```diff
@@ -16,13 +19,52 @@ class Thurloe extends Actor with MyService {
   // this actor only runs our route, but you could add
   // other things here, like request stream processing
   // or timeout handling
-  def receive = runRoute(myRoute)
+  def receive = runRoute(thurloeRoutes)
 }
 
 
 // this trait defines our service behavior independently from the service actor
+@Api(value="/thurloe", description = "Thurloe service", produces = "application/json", position = 1)
 trait MyService extends HttpService {
 
+  import spray.httpx.SprayJsonSupport.sprayJsonUnmarshaller
+
+  def thurloeRoutes: Route =
+  pathPrefix("thurloe") {
+    path(Segment / "get" / Segment) { (userId, key) =>
+      get {
+        respondWithMediaType(`application/json`)
+        complete {
+          // TODO: Get a value from the DB instead of this placeholder
+          val json = KeyValuePair(key, "Bob Loblaw's Law Blog").toJson
+          json.toString
```

Comment:
OCD: IntelliJ made this yellow because it wants `toString()` instead of `toString`. Any strong objections to not-pleasing the IDEA gods?

Also, burying the lede, I'm used to `prettyPrint` instead of `toString()` anyways, or explicitly calling the `compactPrint` if we somehow need to save on bytes.